src/app/core/record-countries/record-countries.service.ts

```diff
@@ -29,25 +30,7 @@ export class RecordCountriesService {
   ) {}
 
   getCountryCodes() {
-    if (!this.$countryCodes) {
-      this.$countryCodes = new ReplaySubject(1)
-      this._http
-        .get<RecordCountryCodesEndpoint>(
-          environment.API_WEB + `countryNamesToCountryCodes.json`,
-          {
-            headers: this.headers,
-          }
-        )
-        .pipe(
-          retry(3),
-          catchError((error) => this._errorHandler.handleError(error)),
-          tap((value) => {
-            this.$countryCodes.next(value)
-          })
-        )
-        .subscribe()
-    }
-    return this.$countryCodes.asObservable()
+    return of(countryCodes)
```

Comment:
@leomendoza123 Could you implement a sort method, also the properties with all country translations are missing